api/src/main/java/org/eclipse/microprofile/config/spi/ConfigBuilder.java

```diff
@@ -65,6 +65,14 @@
      */
     ConfigBuilder addDiscoveredConverters();
 
+    /**
+     * Add all interceptors which can be <a href="ConfigSourceInterceptor.html#discovery">discovered</a> from
+     * this configuration builder's {@linkplain #forClassLoader(ClassLoader) class loader}.
+     *
+     * @return this configuration builder instance
+     */
+    ConfigBuilder addDiscoveredInterceptors();
```

Comment:
This method should be deleted as the inteceptor bit has been lifted out.

Reply:
Yes, I've missed that one. Removing.